packages/terra-aggregator/tests/wdio/aggregator-spec.js

```diff
@@ -1,54 +1,36 @@
 Terra.describeViewports('Aggregator', ['large'], () => {
-  describe('Standalone', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-aggregator/aggregator/aggregator-standalone');
-
-      browser.click('#test-aggregator #section1');
-      browser.click('#test-aggregator #section2');
-      browser.click('#test-aggregator #section3');
-    });
-
-    Terra.it.validatesElement();
+  it('displays standalone aggregator', () => {
+    browser.url('/#/raw/tests/terra-aggregator/aggregator/aggregator-standalone');
```

Comment:
```suggestion
    browser.url('/raw/tests/terra-aggregator/aggregator/aggregator-standalone');
```

Any place `/#/` is used we need to get cleaned up to prevent re-directs when test pages are loaded. Terra-toolkit will wait for it to redirect but it's slower.